components/atoms/UserCard/user-card.tsx

```diff
@@ -18,13 +18,13 @@ const UserCard = ({ username, name, meta, loading }: UserCardProps) => {
   const avatarUrl = getAvatarByUsername(username);
 
   return (
-    <div className="w-full pb-6 border rounded-lg bg-light-slate-1 border-zinc-200">
+    <div className="pb-6 border bg-light-slate-1 w-max rounded-xl border-zinc-200">
```

Comment:
```suggestion
    <div className="pb-6 border bg-light-slate-1 w-full rounded-lg border-zinc-200">
```

Return the full width for the `user-card` and the `border-radius` component.

#### Why though!?

- The width `w-full` for size inheritance reason
- The Border radius `rounded-lg` for uniformity reason, looking at all other sibling components/divs on the `Right (Filter) Div` and `Left (userCard) div`.